castai/resource_node_template.go

```diff
@@ -54,6 +55,7 @@ const (
 	FieldNodeTemplateSpotInterruptionPredictionsEnabled       = "spot_interruption_predictions_enabled"
 	FieldNodeTemplateSpotInterruptionPredictionsType          = "spot_interruption_predictions_type"
 	FieldNodeTemplateStorageOptimized                         = "storage_optimized"
+	FieldNodeTemplateStorageOptimizedStatus                   = "storage_optimized_status"
```

Comment:
Could we please change the name? May be it is just me but I find that it sounds like a read-only attribute, not the value to set. Like in kubernetes status is not something to be set by user but only to be read.
```suggestion
	FieldNodeTemplateStorageOptimizedState                    = "storage_optimized_state"
```

Reply:
Hmm, do you have a proposal for this? I couldn't come up with a better name